### ISSUE TITLE
Python3.11 support

### ISSUE THREAD

**Issue by Rogalek** [aws-cloudformation/cfn-lint#2788]
### Describe the feature you'd like to request

Could you please add python 3.11?

```bash
AWS CloudFormation Linter................................................Failed
- hook id: cfn-python-lint
- exit code: 2

E3030 You must specify a valid value for Runtime (python3.11). Valid values are ["dotnet6", "dotnetcore1.0", "dotnetcore2.0", "dotnetcore2.1", "dotnetcore3.1", "go1.x", "java11", "java17", "java8", "java8.al2", "nodejs", "nodejs10.x", "nodejs12.x", "nodejs14.x", "nodejs16.x", "nodejs18.x", "nodejs4.3", "nodejs4.3-edge", "nodejs6.10", "nodejs8.10", "provided", "provided.al2", "python2.7", "python3.10", "python3.6", "python3.7", "python3.8", "python3.9", "ruby2.5", "ruby2.7", "ruby3.2"]
template.yaml:6:3
```

### Describe the solution you'd like

Adding python3.11 to not show error when Runtine: 3.11

### Is this something that you'd be interested in working on?

- [x] 👋 I may be able to implement this feature request

### Would this feature include a breaking change?

- [ ] ⚠️ This feature might incur a breaking change

**Comment by kddejong** [comment]
The values for that are brought from https://github.com/boto/botocore/blob/99d6a50ea6c2bf6e78e9dbe8dcb1e23e391f3fc1/botocore/data/lambda/2015-03-31/service-2.json#L5199C1-L5234C1

I don't see any announcements for Python 3.11 support.  Do you have one available?

**Comment by Rogalek** [comment]
Ok I see, thank you.

I will add issue in their repo too: https://github.com/boto/botocore/issues/2980

**Comment by Rogalek** [comment]
They are working on it here: https://github.com/aws/aws-lambda-base-images/issues/62

**Comment by Rogalek** [comment]
Its published: https://github.com/aws/aws-lambda-base-images/issues/62#issuecomment-1635917903

**Comment by kddejong** [comment]
Now included in the automated maintenance task

https://github.com/aws-cloudformation/cfn-lint/pull/2804/files#diff-a5a6f378b19287fd419ef6c9edf44d4493163add0a727420af026573a02f165aR240311

**Comment by whoDoneItAgain** [comment]
looks like they've released the 3.11 runtime and its included in the 132.0.0 spec.

**Comment by kddejong** [comment]
The newest release has the 132.0.0 spec and has python 3.11 in it.
